File: VB/Series_3DLineChart/Form1.vb
Imports System
Imports System.Windows.Forms
Imports DevExpress.XtraCharts
' ...

Namespace Series_3DLineChart
    Partial Public Class Form1
        Inherits Form

        Public Sub New()
            InitializeComponent()
        End Sub

        Private Sub Form1_Load(ByVal sender As Object, ByVal e As EventArgs) Handles MyBase.Load
            ' Create a new chart.
            Dim LineChart3D As New ChartControl()

            ' Add a line series to it.
            Dim series1 As New Series("Series 1", ViewType.Line3D)

            ' Add points to the series.
            series1.Points.Add(New SeriesPoint("A", 12))
            series1.Points.Add(New SeriesPoint("B", 4))
            series1.Points.Add(New SeriesPoint("C", 17))
            series1.Points.Add(New SeriesPoint("D", 7))
            series1.Points.Add(New SeriesPoint("E", 12))
            series1.Points.Add(New SeriesPoint("F", 4))
            series1.Points.Add(New SeriesPoint("G", 17))
            series1.Points.Add(New SeriesPoint("H", 7))

            ' Add both series to the chart.
            LineChart3D.Series.Add(series1)


            ' Customize the view-type-specific properties of the series.
            Dim myView As Line3DSeriesView = CType(series1.View, Line3DSeriesView)
            myView.LineWidth = 5
            myView.LineThickness = 4

            ' Access the diagram's options.
            CType(LineChart3D.Diagram, XYDiagram3D).ZoomPercent = 110
            CType(LineChart3D.Diagram, XYDiagram3D).VerticalScrollPercent = 10

            ' Add a title to the chart and hide the legend.
            Dim chartTitle1 As New ChartTitle()
            chartTitle1.Text = "3D Line Chart"
            LineChart3D.Titles.Add(chartTitle1)
            LineChart3D.Legend.Visibility = DevExpress.Utils.DefaultBoolean.False

            ' Add the chart to the form.
            LineChart3D.Dock = DockStyle.Fill
            Me.Controls.Add(LineChart3D)
        End Sub

    End Class
End Namespace

File: CS/Series_3DLineChart/Form1.cs
using System;
using System.Windows.Forms;
using DevExpress.XtraCharts;
// ...

namespace Series_3DLineChart {
    public partial class Form1 : Form {
        public Form1() {
            InitializeComponent();
        }

        private void Form1_Load(object sender, EventArgs e) {
            // Create a new chart.
            ChartControl LineChart3D = new ChartControl();

            // Add a line series to it.
            Series series1 = new Series("Series 1", ViewType.Line3D);

            // Add points to the series.
            series1.Points.Add(new SeriesPoint("A", 12));
            series1.Points.Add(new SeriesPoint("B", 4));
            series1.Points.Add(new SeriesPoint("C", 17));
            series1.Points.Add(new SeriesPoint("D", 7));
            series1.Points.Add(new SeriesPoint("E", 12));
            series1.Points.Add(new SeriesPoint("F", 4));
            series1.Points.Add(new SeriesPoint("G", 17));
            series1.Points.Add(new SeriesPoint("H", 7));

            // Add both series to the chart.
            LineChart3D.Series.Add(series1);


            // Access the series options.
            series1.Label.PointOptions.PointView = PointView.ArgumentAndValues;

            // Customize the view-type-specific properties of the series.
            Line3DSeriesView myView = (Line3DSeriesView)series1.View;
            myView.LineWidth = 5;
            myView.LineThickness = 4;

            // Access the diagram's options.
            ((XYDiagram3D)LineChart3D.Diagram).ZoomPercent = 110;
            ((XYDiagram3D)LineChart3D.Diagram).VerticalScrollPercent = 10;

            // Add a title to the chart and hide the legend.
            ChartTitle chartTitle1 = new ChartTitle();
            chartTitle1.Text = "3D Line Chart";
            LineChart3D.Titles.Add(chartTitle1);
            LineChart3D.Legend.Visible = false;

            // Add the chart to the form.
            LineChart3D.Dock = DockStyle.Fill;
            this.Controls.Add(LineChart3D);
        }

    }
}